mesh/mesh.cpp

```diff
@@ -761,6 +761,10 @@ const GeometricFactors* Mesh::GetGeometricFactors(const IntegrationRule& ir,
       GeometricFactors *gf = geom_factors[i];
       if (gf->IntRule == &ir && (gf->computed_factors & flags) == flags)
       {
+         if (gf->RecomputeStatus())
+         {
+            gf->Recompute();
```

Comment:
@v-dobrev , in my application if I call ``->Recompute()`` It breaks my baselines. However; if I call ``AssembleWithNodes(...)`` then I get the correct answer. I'm wondering if the nodes used to create the class are no longer valid?